### PR TITLE
Adds frame-aware yielding example.

### PR DESCRIPTION
**Pull request by malchata**
Changes proposed in this pull request:

- @mmocy consulted with me on another yielding strategy which nests a `setTimeout` call within a `requestAnimationFrame` call if `isInputPending` is `true`. This PR updates the optimize long tasks guide with that example.

### CODE REVIEW

File: src/site/content/en/blog/optimize-long-tasks/index.md [hunk 1]
```diff
@@ -294,6 +294,70 @@ async function saveSettings () {
 
 With this approach, you get a fallback for browsers that don't support `isInputPending()` by using a time-based approach that uses (and adjusts) a deadline so that work will be broken up where necessary, whether by yielding to user input, or by a certain point in time.
 
+### Frame-aware yielding
+
+Now for a more nuanced approach, which is something you could call "frame-aware" yielding. This yielding technique is a bit different than the ones outlined previously. Yet, the effects of it are more aware of when a frame needs to be presented, and does its best to yield when this is the case:
+
+```js
```
Comment (philipwalton) [inline src/site/content/en/blog/optimize-long-tasks/index.md:301]: Out of curiosity, why did you change this from the original [strawman](https://gist.github.com/mmocny/d54c381d46e69fb7d1337a4bd566f6c7) @mmocny proposed? Was this something you guys discussed in the meeting?

**Comment by malchata** [comment]
I was attempting to remove the module global and it was a bad idea.

File: src/site/content/en/blog/optimize-long-tasks/index.md (hunk 1 above)
Comment (mmocny) [reply src/site/content/en/blog/optimize-long-tasks/index.md:301]: FWIW I'm adding a hopefully better strawman (but still strawman) into my lightning talk from perf.now(), will likely publish today so we can look at it.
